requirements.txt

```diff
@@ -0,0 +1,10 @@
+lxml
+lockfile
+paramiko
+pbr
```

Comment:
[`dulwich`](https://github.com/lago-project/lago/blob/adding_requirements.txt/scripts/version_manager.py#L10) (used in  `scripts/version_manager.py` which is called by [`setup.py`](https://github.com/lago-project/lago/blob/adding_requirements.txt/setup.py#L37)), [`file-magic`](https://github.com/lago-project/lago/blob/adding_requirements.txt/lago/templates.py#L26) and `libvirt-python` are missing.